aws_lambda_builders/actions.py

```diff
@@ -159,3 +162,26 @@ def execute(self):
             new_destination = os.path.join(self.dest_dir, name)
 
             shutil.move(dependencies_source, new_destination)
+
+
+class CleanUpAction(BaseAction):
```

Comment:
I assume this will remove all the files in a directory but doesn't delete the directory itself right?
If so, can we add some comments to describe this behavior.